test/support/assets/migrations/20111117063700-createPerson.coffee

```diff
@@ -7,8 +7,8 @@ module.exports =
           type: DataTypes.BOOLEAN
           defaultValue: false
           allowNull: false
-    .complete done
+    .done () -> done()
```

Comment:
could also use `nodeify` to keep the old way of doing it, the new way won't capture errors